src/service/map.item.service.ts

```diff
@@ -27,10 +29,12 @@ export const mapItem = (item: Item, deliveryDetails: DeliveryDetails | undefined
                 .getCheckDetailsItem({companyName: item?.companyName, companyNumber: item?.companyNumber, itemOptions, deliveryDetails});
         } else {
             const deliveryMethod = MapUtil.mapDeliveryMethod(item?.itemOptions);
+            const emailCopyRequired = MapUtil.mapEmailCopyRequired(itemOptions);
             const address = MapUtil.mapDeliveryDetails(deliveryDetails);
             const certificateDetails = {
                 certificateType: MapUtil.mapCertificateType(itemOptions.certificateType)
             };
+       const whatHappensNextText = itemOptions.deliveryTimescale === "same-day" ? SAME_DAY_HAPPENS_NEXT_TEXT : DEFAULT_TEXT;
```

Comment:
shouldn't this be indented further to line up with the rest of the else statement contents?

Reply:
I'll adjust, thanks.

Reply:
Done